win32ss/user/ntuser/winpos.c

```diff
@@ -2048,7 +2048,7 @@ co_WinPosSetWindowPos(
        * change.
        */
       if ( ( VisBefore != NULL &&
-             VisAfter != NULL &&
+             VisAfter != NULL && !(WvrFlags & WVR_REDRAW) &&
             !(WinPos.flags & SWP_NOCOPYBITS) &&
             ((WinPos.flags & SWP_NOSIZE) || !(WvrFlags & WVR_REDRAW)) &&
```

Comment:
Since now the test condition has been modified so that we _**require**_ always that `!(WvrFlags & WVR_REDRAW)` be true, this whole other line 2053 (and so, the `WinPos.flags & SWP_NOSIZE` test) now appears to be redundant.
Indeed, because `!(WvrFlags & WVR_REDRAW)` is now going to be always true, `(anything || TRUE)` will always be true.

Reply:
Simplified logic based on comments. Thanks.